src/components/view/ContactInfo/contactInfoEventsCard.js

```diff
@@ -0,0 +1,65 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+
+import { Card, CardHeader, CardText } from 'material-ui/Card';
+import { List, ListItem } from 'material-ui/List';
+
+import { getImageForEnv } from 'static/images/index'
+
+class ContactInfoEventsCard extends Component {
+
+    formatUidToDate = uid => {
+        const splitUid = uid.split('');
+        return `${splitUid.slice(4,6).join('')}/${splitUid.slice(7,9).join('')}/${splitUid.slice(0,4).join('')}`
+    }
+
+    render() {
+        const { events } = this.props
+
+        const cardProps = {
+            initiallyExpanded: true
+        }
+
+        const cardHeaderProps = {
+            title: 'Events',
+            actAsExpander: true,
+            showExpandableButton: true,
+            titleColor: this.props.titleColor
+        }
+
+        const cardTextProps = {
+            expandable: true
+        }
+
+        const noEventsText = 'There are no events for this user'
```

Comment:
Since this is such a short piece of text, why not put it in the JSX directly?

#question #recommended

Reply:
we've been doing localization work and so I've been struggling with combing through our frontend app looking for hard-coded strings that need to be pulled out and translated. however I agree we don't really need to worry about that here